gqlgen/graph/schema.resolvers.go

```diff
@@ -22,13 +22,10 @@ func (r *mutationResolver) CreateQuote(ctx context.Context, input model.NewQuote
 		Quote:  input.Quote,
 		Author: input.Author,
 	}
-
 	byteArray, _ := json.Marshal(quote)
 	buffer := bytes.NewBuffer(byteArray)
-
 	request, _ := http.NewRequest("POST", "http://34.160.33.1:80/quotes", buffer)
 	request.Header.Set("x-api-key", fmt.Sprint(ctx.Value("myKey")))
-
 	client := &http.Client{}
 	response, _ := client.Do(request)
```

Comment:
There are a few instances in your code where you neglect to handle potential errors. Most, if not all, don't really matter given that the error is probably handled by your REST API that is also under your control. In the event that you were using someone else's API, they might be good errors to handle in your own API layer.